client/store/index.js

```diff
@@ -3,8 +3,9 @@ import {createLogger} from 'redux-logger'
 import thunkMiddleware from 'redux-thunk'
 import {composeWithDevTools} from 'redux-devtools-extension'
 import user from './user'
+import users from './users'
 
-const reducer = combineReducers({user})
+const reducer = combineReducers({user, users})
```

Comment:
I think having `user` and `users` is a bit confusing.  Can we rename one or both of these ?